src/test/suite/fixtures/recorded/scopes/token/ditchFinePastBat.yml

```diff
@@ -0,0 +1,37 @@
+languageId: plaintext
+command:
+  spokenForm: ditch fine past bat
+  version: 3
+  targets:
+    - type: range
+      anchor:
+        type: primitive
+        mark: {type: decoratedSymbol, symbolColor: default, character: f}
+      active:
+        type: primitive
+        mark: {type: decoratedSymbol, symbolColor: default, character: b}
+      excludeAnchor: false
+      excludeActive: false
+  usePrePhraseSnapshot: true
+  action: {name: remove}
+initialState:
+  documentContents: |2
+        foo
+        bar.
+  selections:
+    - anchor: {line: 2, character: 0}
+      active: {line: 2, character: 0}
+  marks:
+    default.f:
+      start: {line: 0, character: 4}
+      end: {line: 0, character: 7}
+    default.b:
+      start: {line: 1, character: 4}
+      end: {line: 1, character: 7}
+finalState:
+  documentContents: |2
+        .
```

Comment:
This test was broken in your version, because you were looking to see if the end of the line containing the start of the range contained the end of the range, rather than the line containing the end of the range contained the end of the range